platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -4,39 +4,65 @@ declare -r SCRIPT_NAME="$(basename "$0")"
 declare -r SCRIPT_PATH="$(readlink -f "$0")"
 declare -r SCRIPT_DIR="$(dirname "$SCRIPT_PATH")"
 
+declare -r YES_PARAM="yes"
+declare -r NO_PARAM="no"
+
+declare -r VERBOSE_ERROR="1"
+declare -r VERBOSE_WARNING="2"
+declare -r VERBOSE_NOTICE="3"
+declare -r VERBOSE_INFO="4"
+
+declare -r VERBOSE_MAX="${VERBOSE_INFO}"
+declare -r VERBOSE_MIN="${VERBOSE_ERROR}"
+
 declare -r EXIT_SUCCESS="0"
-declare -r EXIT_ERROR="1"
+declare -r EXIT_FAILURE="1"
```

Comment:
why failure and not error? what is the idea behind the name change?

Reply:
@liat-grozovik just a name convention. Nothing special.

Reply:
i think we should continue with error. this is more convention indication for error